sql/database.sql

```diff
@@ -8294,3 +8294,13 @@ CREATE TABLE `immunization_observation` (
   PRIMARY KEY (`imo_id`)
```

Comment:
Should modify the Administration->Lists gui script to show a nice title for the notes column when a user edits this list:
https://github.com/openemr/openemr/blob/master/interface/super/edit_list.php#L933-L960